services/OrderService/Repository/OrderRepository.cs

```diff
@@ -1,37 +1,53 @@
+using Microsoft.EntityFrameworkCore;
+using OrderService.Data;
 using OrderService.Interface;
 using OrderService.Models;
 
 namespace OrderService.Repository;
 
 public class OrderRepository : IOrderRepository
 {
-    public Task<Order> CreateAsync(Order orderModel)
+    private readonly InMemoryContext _context;
+
+    public OrderRepository(InMemoryContext context)
     {
-        throw new NotImplementedException();
+        _context = context;
     }
 
-    public Task<Order> DeleteAsync(int id)
+    public async Task CreateAsync(Order orderModel)
     {
-        throw new NotImplementedException();
+        await _context.AddAsync(orderModel);
     }
 
-    public Task<bool> Exists(int id)
+    public async Task DeleteAsync(Order order)
     {
-        throw new NotImplementedException();
+        _context.Remove(order);
     }
 
-    public Task<List<Order>> GetAllAsync()
+    public async Task<bool> Exists(int id)
     {
-        throw new NotImplementedException();
+        var order = await _context.Orders.FirstAsync(x => x.Id == id);
+        return order == null ? true : false;
```

Comment:
Looks like the same error as the previous one, please correct this is sending true when it doesn't exist and sending false when it does exist.